core/services/keeper/upkeep_executer_test.go

```diff
@@ -144,9 +148,60 @@ func Test_UpkeepExecuter_PerformsUpkeep_Happy(t *testing.T) {
 		txm.AssertExpectations(t)
 	})
 
+	t.Run("runs upkeep on triggering block number on non-EIP1559 chain", func(t *testing.T) {
+		db, config, ethMock, executer, registry, upkeep, job, jpv2, txm, _, _, _ := setup(t)
+
+		config.Overrides.GlobalEvmEIP1559DynamicFees = null.BoolFrom(false)
```

Comment:
the two tests look the same to me with the change of this config. Could you parameterise the two tests to share the code. (e.g. https://github.com/smartcontractkit/chainlink/blob/develop/core/cmd/remote_client_test.go#L151-L162)

Reply:
Well, it is kind of difficult to refactor all tests to the test-table approach. I've updated it to use the same function but with a different flag. Check again.